.github/workflows/graalpy-install-package.yml

```diff
@@ -0,0 +1,40 @@
+name: graalpy-install-package
+on:
+  push:
+    paths:
+      - 'graalpy-install-package/**'
+      - '.github/workflows/graalpy-install-package.yml'
+  # pull_request: (requires EE, which is unavailable in PRs)
+  schedule:
+    - cron: "0 0 1 * *" # run every month
```

Comment:
The versions are locked, so I'm not sure we need a cronjob for this (it doesn't run on an EA or dev build)